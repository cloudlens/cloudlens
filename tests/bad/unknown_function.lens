stream when (e){
    unknown_function(e)
}
