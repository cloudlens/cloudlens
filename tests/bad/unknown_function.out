Error: tests/bad/unknown_function.lens, block starting line 1:
<eval>:2 TypeError: unknown_function is not a function
